kustomization.yaml

```diff
@@ -1,7 +1,8 @@
 bases:
 - ambassador/base
 - argo/base
-- github.com/swiftdiaries/kubeflow//components/profile-controller/config?ref=profiles-controller
+- jupyter-web-app/base
+#- github.com/swiftdiaries/kubeflow//components/profile-controller/config/default?ref=profiles-controller
```

Comment:
should we remove this comment?

Reply:
It's currently failing because remote builds don't work well with relative paths like `../`
We need pull the config/ folder of the profiles-controller from kfctl to enable this so I thought it'd be commented till then